.ci/auto-sign-zip

```diff
@@ -19,6 +19,9 @@ root="$( cd -P "$( dirname "$csource" )/../" && pwd )"
 #       which allows it to properly load user plugins.
 macho_entitled_binary_names=( "ruby" )
 
+# Maximum number of background jobs to allow
+max_jobs="100"
```

Comment:
100 seems like a lot! Any reason behind choosing this number?

Reply:
Just the want for it to complete faster. If it ends up being too aggressive it'll be easy to back off (just adjust this down to like 50, 20, 10, or w/e)